internal/services/applications/application_resource_test.go

```diff
@@ -1572,3 +1594,20 @@ resource "azuread_application" "test" {
 }
 `, data.RandomInteger)
 }
+
+func (r ApplicationResource) password(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+provider "azuread" {}
+
+data "azuread_client_config" "current" {}
+
+resource "azuread_application" "test" {
+  display_name = "acctest-APP-%[1]d"
+  owners = [ data.azuread_client_config.current.object_id ]
+
+  password {
+    display_name = "acctest-APP-%[1]d"
+  }
+}
+`, data.RandomInteger)
+}
```

Comment:
Configuration for the additional test

```suggestion
}

func (r ApplicationResource) passwordComplete(data acceptance.TestData, startDate, endDate string) string {
	return fmt.Sprintf(`
provider "azuread" {}

data "azuread_client_config" "current" {}

resource "azuread_application" "test" {
  display_name = "acctest-APP-%[1]d"
  owners       = [data.azuread_client_config.current.object_id]

  password {
    display_name = "acctest-appPasswordComplete-%[1]d"
    start_date   = "%[2]s"
    end_date     = "%[3]s"
  }
}
`, data.RandomInteger, startDate, endDate)
}
```